build.sbt

```diff
@@ -44,11 +44,13 @@ ThisBuild / scmInfo := Some(
 
 ThisBuild / crossScalaVersions := Seq("2.11.12", "2.12.16", "2.13.7", "3.1.3")
 
-ThisBuild / githubWorkflowBuildPreamble ++= Seq(
-  WorkflowStep.Use(
+val setupSbt =   WorkflowStep.Use(
     UseRef.Public("sbt", "setup-sbt", "v1"),
     name = Some("Setup sbt")
-  ),
+  )
```

Comment:
Is the indentation intended this way? I would expect something like
```scala
val setupSbt = WorkflowStep.Use(
  UseRef.Public("sbt", "setup-sbt", "v1"),
  name = Some("Setup sbt")
)
```
instead, no?

Reply:
Sorry, I clearly struggle without auto-formatting 😂